### PR TITLE
mep12 on plotfile_demo.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/plotfile_demo.py [hunk 1]
```diff
@@ -1,35 +1,49 @@
...
-plotfile(fname2, cols=(0, 1), delimiter=' ')
-plotfile(fname2, cols=(0, 2), newfig=False, delimiter=' ')  # use current figure
-gca().set_xlabel(r'$x$')
-gca().set_ylabel(r'$f(x) = x^2, x^3$')
+plt.plotfile(fname2, cols=(0, 1), delimiter=' ')
+plt.plotfile(
+    fname2,
+    cols=(
+        0,
+        2),
```
Comment (thisch) [inline examples/pylab_examples/plotfile_demo.py]: This looks a bit weird. Please put the tuple on the same line as the cols kwarg, i.e., ``cols=(0, 2),``.

**Comment by WeatherGod** [comment]
PEP8:
```
/home/travis/build/matplotlib/matplotlib/examples/pylab_examples/plotfile_demo.py:38:12: E231 missing whitespace after ','
```

**Comment by ericmjl** [comment]
Anything else needed?

File: examples/pylab_examples/plotfile_demo.py [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
 
 # test 3; use semilogy for volume
-plotfile(fname, ('date', 'volume', 'adj_close'), plotfuncs={'volume': 'semilogy'})
+plt.plotfile(
+    fname,
+    ('date',
+     'volume',
+     'adj_close'),
+    plotfuncs={
+        'volume': 'semilogy'})
```
Comment (thisch) [inline examples/pylab_examples/plotfile_demo.py]: 7 lines for the ``plot_file`` function is a bit too much. Please put it in 2 lines. 

```
plt.plotfile(fname, ('date', 'volume', 'adj_close'), 
             plotfuncs={'volume': 'semilogy'})
```

File: examples/pylab_examples/plotfile_demo.py [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
-plotfile(fname2, cols=(0, 1), delimiter=' ')
-plotfile(fname2, cols=(0, 2), newfig=False, delimiter=' ')  # use current figure
-gca().set_xlabel(r'$x$')
-gca().set_ylabel(r'$f(x) = x^2, x^3$')
+plt.plotfile(fname2, cols=(0, 1), delimiter=' ')
+plt.plotfile(
+    fname2,
+    cols=(0, 2),
+    newfig=False,
+    delimiter=' ')  # use current figure
```
Comment (thisch) [inline examples/pylab_examples/plotfile_demo.py]: same here

File: examples/pylab_examples/plotfile_demo.py (hunk 2 above)
Comment (ericmjl) [inline examples/pylab_examples/plotfile_demo.py]: Noted. Will do it for lines 36-40 as well.

**Comment by ericmjl** [comment]
Okay, all done :)

**Comment by tacaswell** [comment]
I did not know that the `plotfile` function existed....

**Comment by ericmjl** [comment]
Haha, I didn't either...

**Comment by ericmjl** [comment]
Looks like this one passed. Merge or delete? Sorry for bugging, I am just trying to keep the pace going; my workflow is 3 `MEP12` changes at any time so I don't get too confused.

**Comment by jenshnielsen** [comment]
Maybe we should get rid of the plotfile function. But as long as it's there I think it's fine to keep the example

**Pull request merged**